docs/getting-started/configuration.md

```diff
@@ -0,0 +1,307 @@
+---
+id: index
+title: Getting Started, configuring Backstage
+description: Getting started with your initial Backstage configuration
+---
+
+This is part two of the Getting Started documentation of Backstage. The steps in
+this tutorial assume you've installed Backstage app from the npm repository,
+like in the [Getting Started guide](./index.md) and want to configure Backstage.
+
+At the end of this tutorial, you can expect:
+
+- Backstage to use a PostgreSQL database
+- You'll authenticate using one of the auth providers
+- The Backstage GitHub integration to be configured
+- You're able to use Software Templates
+
+### Prerequisites
+
+- Access to a Linux-based operating system, such as Linux, MacOS or
+  [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/)
+- An account with elevated rights to install prerequisites
+- If the database is not hosted on the same server as the Backstage app, the
+  PostgreSQL port needs to be accessible (the default is 5432 or 5433)
+
+### Install and configure PostgreSQL
```

Comment:
For Mac users, https://postgresapp.com is a single click installation and works automagically. Probably worth adding here along with the apt-get based installation :)

Reply:
Yep, It's listed in the installation options for Mac on the official site.